tests/test_pipelineconfig.py

```diff
@@ -39,3 +39,9 @@ def test_config_loading(path: str):
         ),
         corpusprocessing=CorpusProcessingConfig(),
     )
+
+
+def test_update_nested_dict():
+    some_dict = {"a": {"b": 1}}
+    PipelineConfig.update_nested_dict(some_dict, "a.b", 2)
+    assert some_dict["a"]["b"] == 2
```

Comment:
Not sure what this is for?

Reply:
The test or the method? The method sets or updates values in a dict, in this case the TOML-dict before stuffed in to the `Config` objects. See `conspiracies/pipeline/config:PipelineConfig.from_toml_file()`.

Reply:
ahh I see